packages/gamut/src/NotificationListNew/assets/emptyNotificationMessages.tsx

```diff
@@ -0,0 +1,41 @@
+import beeGraphic from './bee.svg';
+import browserGraphic from './browser.svg';
+import confettiGraphic from './confetti.svg';
+import plantGraphic from './plant.svg';
+import sunGraphic from './sun.svg';
+
+export const emptyNotificationMessages = [
+  {
+    text: "You're all caught up!",
+    subtext: 'Go forth and learn new things.',
+    graphic: 'bee',
+  },
+  {
+    text: "You're all caught up!",
+    subtext: 'Time to learn new things.',
+    graphic: 'confetti',
+  },
+  {
+    text: "You're all caught up!",
+    subtext: 'The future looks bright.',
+    graphic: 'sun',
+  },
+  {
+    text: "You're up to date!",
+    subtext: 'Go forth and learn new things.',
+    graphic: 'plant',
+  },
+  {
+    text: "You're up to date!",
+    subtext: 'Clear future ahead.',
+    graphic: 'browser',
+  },
+];
+
+export const slugToSvgMap: Record<string, string> = {
```

Comment:
Nit: you should be able to get away with not including this:

```suggestion
export const slugToSvgMap = {
```

If you use an `as const` on `emptyNotificationMessages`. I think that'd be a little more preferable because it's more representative of how the types work. https://blog.logrocket.com/const-assertions-are-the-killer-new-typescript-feature-b73451f35802/

Meaning: `slugToSvgMap` is really only supposed to be used with one of these known keys (`'bee'`, `'confetti'`, ...). `Record<string, string>` says it will give back a `string` value under _any_ key.

Definitely not a high priority change request 😄, just a nice-to-have.

Reply:
ooh thanks for teaching me!